sqlmesh/core/loader.py

```diff
@@ -445,5 +448,6 @@ def _model_cache_entry_id(self, model_path: Path) -> str:
                 [
                     str(int(max(m for m in mtimes if m is not None))),
                     self._loader._context.config.fingerprint,
+                    self._loader._context.default_catalog or "",
```

Comment:
Isn't it captured in the config's fingerprint?

Reply:
_Maybe_. Depends on how Airflow gets implemented right? If that is something that is configured outside of the project config (for example in the plugin python script) then this would be needed. I see though how this shouldn't be needed for the current engine adapter related approach.

Reply:
Worth leaving a comment